pkg/agent/util/dump_linux.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+
+package util
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/afero"
+	"k8s.io/utils/exec"
+
+	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsctl"
+)
+
+type dumper struct {
+	factory  ofctlClientFactory
+	fs       afero.Fs
+	executor exec.Interface
+}
+
+func (d *dumper) DumpLog(basedir string) error {
+	f := func(targetDir string, logDir string, prefixFilter string) error {
```

Comment:
Is it possible to move line 40 - 69 to a separate func in pkg/util/dump.go, and let it shared by agent and controller?

Reply:
Updated, PTAL.